src/hash_table.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+from typing import Union
+import random
+
+
+class Node:
+    def __init__(self, key: str, val: Union[str, float], next: Node = None):
+        self.key = key
+        self.val = val
+        self.next = next
+
+
+class ChainHashTable:
+    def __init__(self, n: int):
+        self.__bucket = [Node(None, None)] * n
+
+    def insert(self, key: str, val: Union[str, float]) -> bool:
```

Comment:
This method will always return `True`. I think we don't need to return any value from this method.